spec/steps/claim_details/form_spec.rb

```diff
@@ -62,47 +74,77 @@
 
       context 'time_spent has a value in the database' do
         let(:time_spent_in_db) { 100 }
+        let(:work_before_date) { Date.new(2023, 1, 1) }
+        let(:work_after_date) { Date.new(2023, 1, 1) }
 
         it 'clears the database field' do
           expect(subject.save).to be_truthy
           expect(application.reload).to have_attributes(
-            time_spent: nil
+            time_spent: 160
```

Comment:
so you have changed the test results here and I am not 100% sure why. This is a test to confirm the DB field is cleared, but that is not longer the result?